src/palette/qml/MuseScore/Palette/internal/PaletteTree.qml

```diff
@@ -83,7 +83,8 @@ StyledListView {
         }
 
         if (paletteModel) {
```

Comment:
Sorry, a bit late, but is the `if (paletteModel)` check still relevant even though `paletteModel` is not used here?

Reply:
@krasko78 you can remove this check in your next PR

Reply:
I'll have it in mind. Thanks.

Reply:
I took care of this in my next PR [24540](https://github.com/musescore/MuseScore/pull/24540).